spec/rubocop/cop/style/unneeded_ternary_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::UnneededTernary do
+  subject(:cop) { described_class.new }
+
+  it 'registers no offense for vars' do
+    expect_no_offenses('a = b ? d : c')
```

Comment:
Not sure why all the example are using some assignment - obviously the assignment doesn't play any part in what the cop is checking for.